static/src/js/components/Map/Map.jsx

```diff
@@ -184,26 +232,78 @@ const Map = ({
     // Set the new padding value with the new panelsWidth
     const newPadding = [0, 0, 0, panelsWidth]
 
+    // Get the current center longitude and latitude
+    let [currentLongitude, currentLatitude] = view.getCenter()
+    let newCenter = {
+      longitude: currentLongitude,
+      latitude: currentLatitude
+    }
+
     // In order to keep the map from moving when we change the padding we need to adjust
-    // the center longitude of the map. We can calculate the difference in pixels between
-    // the previousPanelsWidth and the panelsWidth and adjust the center longitude by that
-    // difference.
+    // the center of the map.
 
-    // Find longitude difference between the previousPanelsWidth and the panelsWidth
+    // If the current center is 0,0 then use the projection's default center for calculations
+    if (currentLongitude === 0 && currentLatitude === 0) {
+      const projectionConfig = projectionConfigs[projectionCode];
+      [currentLongitude, currentLatitude] = projectionConfig.center
+
+      newCenter = {
+        longitude: currentLongitude,
+        latitude: currentLatitude
+      }
+    }
+
+    // Adjust the center based on the difference in the panels
+
+    // Find the pixel difference between the previousPanelsWidth and the panelsWidth
     const diffInPixels = panelsWidth - previousPanelsWidth
-    // Find the longitude difference in degrees
+
+    // Find the coordinate difference based on the resolution of the view
     const diff = (view.getResolution() * diffInPixels) / 2
 
-    // Get the current center longitude
-    const [currentLongitude, currentLatitude] = view.getCenter()
-    // Generate the new center longitude
-    const newCenter = [currentLongitude + diff, currentLatitude]
+    // If the difference in pixels is not 0, adjust the center of the map
+    if (diffInPixels !== 0) {
```

Comment:
A playwright test to ensure that moving the panel does not change the snapshot might be worth it to ensure this behavior`tests/e2e/panels/panels_behavior.spec.js` and then just the ` await expect(page).toHaveScreenshot('geographic.png', {
          clip: screenshotClip
        })`